sdks/python/src/opik/cli/export_prompt.py

```diff
@@ -0,0 +1,442 @@
+            },
+            "history": [
+                {
+                    "prompt": version.prompt,
+                    "metadata": version.metadata,
+                    "type": version.type if version.type else None,
+                    "commit": version.commit,
+                }
+                for version in prompt_history
+            ],
+            "downloaded_at": datetime.now().isoformat(),
+        }
+
+        # Save to file using the appropriate format
+        if format.lower() == "csv":
+            write_csv_data(prompt_data, prompt_file, prompt_to_csv_rows)
+        else:
+            write_json_data(prompt_data, prompt_file)
+
+        if debug:
+            debug_print(f"Exported prompt: {prompt.name}", debug)
+        return 1
+
+    except Exception as e:
+        console.print(f"[red]Error exporting prompt {prompt.name}: {e}[/red]")
+        return 0
+
+
+def export_prompt_by_name(
+    name: str,
+    workspace: str,
+    output_path: str,
+    max_results: Optional[int],
+    force: bool,
+    debug: bool,
+    format: str,
+) -> None:
+    """Export a prompt by exact name."""
+    try:
+        if debug:
+            debug_print(f"Exporting prompt: {name}", debug)
+
+        # Initialize client
+        client = opik.Opik(workspace=workspace)
+
+        # Create output directory
+        output_dir = Path(output_path) / workspace / "prompts"
+        output_dir.mkdir(parents=True, exist_ok=True)
+
+        if debug:
+            debug_print(f"Target directory: {output_dir}", debug)
+
+        # Try to get prompt by exact name
+        try:
+            prompt = client.get_prompt(name)
+            if not prompt:
+                console.print(f"[red]Prompt '{name}' not found[/red]")
+                return
+
+            if debug:
+                debug_print(f"Found prompt by direct lookup: {prompt.name}", debug)
+        except Exception as e:
+            console.print(f"[red]Prompt '{name}' not found: {e}[/red]")
+            return
+
+        # Export the prompt
+        exported_count = export_single_prompt(
+            client, prompt, output_dir, max_results, force, debug, format
+        )
+
+        # Collect statistics for summary
+        stats = {
+            "prompts": 1 if exported_count > 0 else 0,
+            "prompts_skipped": 0 if exported_count > 0 else 1,
+        }
+
+        # Show export summary
+        print_export_summary(stats, format)
+
+        if exported_count > 0:
+            console.print(
+                f"[green]Successfully exported prompt '{name}' to {output_dir}[/green]"
+            )
+        else:
+            console.print(
+                f"[yellow]Prompt '{name}' already exists (use --force to re-download)[/yellow]"
+            )
+
+    except Exception as e:
+        console.print(f"[red]Error exporting prompt: {e}[/red]")
+        sys.exit(1)
+
+
+def export_experiment_prompts(
+    client: opik.Opik,
+    experiment: Any,
```

Comment:
experiment has a strict type.
If we are not strict with the type hints, we are not only making it harder for other developers to read the code - we also reduce the quality of AI-tools that will work with this code later.